components/sections/Partners.jsx

```diff
@@ -0,0 +1,32 @@
+import Title from '../ui/Title'
+import Image from 'next/image';
+import logo from '../../data/logos'
+const Partners = () => {
+    return (
+        < div className='flex flex-col items-center  justify-center '>
```

Comment:
You should remove empty spaces in the classes

Reply:
done